gc/src/lib.rs

```diff
@@ -508,14 +568,34 @@ impl<T: Trace + ?Sized> GcCell<T> {
                 (*self.cell.get()).root();
             }
 
-            GcCellRefMut {
+            Ok(GcCellRefMut {
                 flags: &self.flags,
                 value: &mut *self.cell.get(),
-            }
+            })
         }
     }
 }
 
+/// An error returned by [`GcCell::try_borrow`](struct.GcCell.html#method.try_borrow).
+#[derive(Debug, Clone, Copy, PartialEq)]
```

Comment:
Eq, Hash, PartialOrd, Ord, Default as well

probably won't need them, doesn't hurt to add. Often it can be an issue when nesting `derive`s